shortcodes.php

```diff
@@ -1055,13 +1055,13 @@ function sc_single_post_meta($atts = Array())
 		<div>
 			<p id="byline">By <?=$byline?></p>
 			<?=$author_title?>
-			<p><?=date('l, F j, Y', strtotime($post->post_date))?></p>
-			<?=$source?>
-			<? if(function_exists('wp_print')) {?>
+			<p><?php echo date( 'l, F j, Y', strtotime( $post->post_date ) ); ?></p>
```

Comment:
We should add the Updated date under the date here too so it displays on non-featured stories

Reply:
Also, if you do end up styling the Updated date for the featured stories, we could do something similar here:
![screen shot 2018-11-08 at 1 46 13 pm](https://user-images.githubusercontent.com/21267784/48220700-0380be00-e35e-11e8-96ef-b4f84d8154f7.png)

Reply:
I'm ok with red; gray or maybe even blue could also work.

Reply:
Yeah, I'm not definitely set with red. I just thought of that first because the example Laura showed us yesterday used red. Blue or grey would work too.